### PR TITLE
Setup build for module package.

### PR DESCRIPTION
**Pull request by josiah-tesfu**
Built the edgepi-rpc-client as a PyPi module. Added workflows for deploying builds, unit and integration testing, and linting.

### CODE REVIEW

File: .github/workflows/pypi-publish.yml
Comment (farzadpanahi) [inline .github/workflows/pypi-publish.yml:1]: there should be a workflow to publish to pypy test on push to staging branch. you should be able to find it from the sdk repo.

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+protobuf==3.20.3
+python-periphery==2.4.1
+pyzmq==25.1.0
+edgepi-python-sdk==1.2.6
```
Comment (farzadpanahi) [inline requirements.txt]: is this the latest sdk ver?

File: tests/integration/conftest.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import subprocess
+import pytest
+
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_and_teardown_server():
+    """ Sets up a server for tests and shut it down when it is finished"""
+    # Run server
+    server_file = os.path.abspath(os.path.join(os.path.dirname(__file__),
```
Comment (farzadpanahi) [inline tests/integration/conftest.py]: you need to fix server setup. there wouldn't be a server file here. you should be installing the server as a pkg.

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

File: tests/__init__.py
Comment (farzadpanahi) [inline tests/__init__.py:1]: add unit tests workflow, even though you don't have any unit tests now.

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

File: tests/integration/conftest.py [hunk 3]
```diff
@@ -8,7 +8,10 @@
 def setup_server():
     "Start server"
     install_cmds = [
-
+        ["wget", "-qO", "-", "https://deb.edgepi.com/edgepi-release.gpg", "|",
+         "sudo", "gpg", "--dearmor", "-o", "/usr/share/keyrings/edgepi-release.gpg"],
+        ["sudo", "tee", "/etc/apt/sources.list.d/edgepi.list", ">/dev/null"],
```
Comment (farzadpanahi) [inline tests/integration/conftest.py]: tha `tee` command is not complete. this is the command you need to run:
```
sudo tee -a /etc/apt/sources.list.d/edgepi.list >/dev/null <<'EOF'
deb [arch=arm64, signed-by=/usr/share/keyrings/edgepi-release.gpg] https://deb.edgepi.com/main bullseye main
deb [arch=arm64, signed-by=/usr/share/keyrings/edgepi-release.gpg] https://deb.edgepi.com/test bullseye test
deb [arch=arm64, signed-by=/usr/share/keyrings/edgepi-release.gpg] https://deb.edgepi.com/nightly bullseye nightly
EOF
```

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

File: tests/integration/conftest.py (hunk 3 above)
Comment (josiah-tesfu) [reply tests/integration/conftest.py]: Could you take a look at the conftest.py code now? The edgepi-rpc-server still isn't located but I don't see any errors in my code.

Comment (josiah-tesfu) [reply tests/integration/conftest.py]: Could you take another look at the file? The edgepi-rpc-server package still isn't located but I don't see any errors in my code.

File: tests/__init__.py
Comment (farzadpanahi) [inline tests/__init__.py:1]: you can add a placeholder test so that unit tests will pass.

**Review by farzadpanahi — DISMISSED** [review]

**Review by farzadpanahi — APPROVED** [review]

**Pull request merged**